hs_file_types/models/base_model_program_instance.py

```diff
@@ -26,7 +25,7 @@ class AbstractModelLogicalFile(AbstractLogicalFile):
 
     # metadata schema (in json format) for model instance aggregation
     # metadata for the model instance aggregation is validated based on this schema
-    metadata_schema_json = JSONField(default=dict)
+    metadata_schema_json = models.JSONField(default=dict)
```

Comment:
we have a bunch of migrations with `default={}` ... example: 
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/hs_file_types/migrations/0006_auto_20170812_1621.py#L140
do we need/want to update these?

Reply:
We don't need to update these old migration files. New migration files have been created.

Reply:
sounds good, thanks